bip-0065.mediawiki

```diff
@@ -271,9 +271,15 @@ semantics and detailed rationale for those semantics.
 https://github.com/petertodd/bitcoin/commit/ab0f54f38e08ee1e50ff72f801680ee84d0f1bf4
 
 
-==Upgrade and Testing Plan==
-
-TBD
+==Deployment==
+
+We reuse the double-threshold switchover mechanism from BIPs 34 and
+66, with the same thresholds, but for nVersion = 4. The new rules are
+in effect for every block (at height H) with nVersion = 4 and at least
+750 out of 1000 blocks preceding it (with heights H-1000..H-1) also
+have nVersion = 4. Furthermore, when 950 out of the 1000 blocks
+preceding a block do have nVersion = 4, nVersion = 3 blocks become
+invalid, and all further blocks enforce the new rules.
```

Comment:
1) We don't lose another bit.

2) The existing IsSuperMajority() mechanism can't do that safely, as it'd mean the soft-fork could be reversed after the fact. The nVersion bits mechanism needs to record state to do this safely.